core/src/nucleus/wasm-test/src/lib.rs

```diff
@@ -51,27 +58,27 @@ pub extern "C" fn debug_hello(encoded_allocation_of_input: usize) -> i32 {
 #[no_mangle]
 pub extern "C" fn debug_multiple(encoded_allocation_of_input: usize) -> i32 {
   let mut mem_stack = SinglePageStack::from_encoded_allocation(encoded_allocation_of_input as u32).unwrap();
-  hdk_debug(&mut mem_stack, "Hello");
-  hdk_debug(&mut mem_stack, "world");
-  hdk_debug(&mut mem_stack, "!");
-  return 0;
+  hdk_debug(&mut mem_stack, &JsonString::from(RawString::from("Hello")));
+  hdk_debug(&mut mem_stack, &JsonString::from(RawString::from("world")));
+  hdk_debug(&mut mem_stack, &JsonString::from(RawString::from("!")));
+  i32::from(RibosomeReturnCode::Success)
```

Comment:
this is just because the fake hdk has no tooling :(

followup is to remove the fake hdks that only exist for testing and do more low level tests and also more tests in the real hdk